docker/terraform/terraform-container.yaml

```diff
@@ -0,0 +1,148 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://cloudify.co/spec/cloudify/5.0.5/types.yaml
+  - plugin:cloudify-docker-plugin
+
+inputs:
+
+  docker_host:
+    type: string
+    required: true
+
+  docker_user:
+    type: string
+    required: true
+
+  terraform_source:
+    type: string
+    default: https://github.com/cloudify-community/terraform-example.git
+
+  container_volume:
+    description: volume that will be mapped to container
+    type: string
+    default: "/terraform"
+
+  terraform_version:
+    description: Terraform version to prepare image for
+    type: string
+    default: "0.11.7"
+
+  terraform_plugins:
+    description: List of Plugins URLs to install on terraform container
+    type: list
+    default:
+      - https://releases.hashicorp.com/terraform-provider-aws/2.49.0/terraform-provider-aws_2.49.0_linux_amd64.zip
+
+  image_tag:
+    description: Tag for the image to build
+    type: string
+    default: "terraform:"
+
+  aws_access_key_id:
+    type: string
+    default: { get_secret: aws_access_key_id }
+
+  aws_secret_access_key:
+    type: string
+    default: { get_secret: aws_secret_access_key }
+
+  aws_default_region:
+    type: string
+    default: { get_secret: ec2_region_name }
```

Comment:
I think that in order to be consistent with our examples we should chage the input name to :"aws_region_name" with default value: "us-east-1"